dist/algoliasearch.helper.js

```diff
@@ -11929,13 +11929,13 @@ SearchParameters.prototype = {
    * @param {value} value value used for filtering
    * @return {SearchParameters}
    */
-  toggleFacetRefinement: function toggleFacetRefinement(facet, value) {
+  _toggleFacetRefinement: function _toggleFacetRefinement(facet, value) {
```

Comment:
Not a big fan of having this function renamed with an underscore and still documenting it. What about renaming it to `toggleConjunctiveFacetRefinement`?